ecl/sss/v1/approval_requests/testing/fixtures.go

```diff
@@ -21,13 +21,22 @@ var listResponse = fmt.Sprintf(`
       "approver_id":"11a98bf9cb144af5a204c9da566d2bd0",
       "request_user_id":"ecid9999888881",
       "service":"network",
-      "action" : [
+      "actions" : [
```

Comment:
Tab and space are mixed here. Can you use only tab?

Reply:
I didn't notice. Fixed.